### PR TITLE
Fix travis 2

### PR DESCRIPTION
**Pull request by adr1ancho**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "ccd-case-activity-web",
```
Comment (tzarsmango) [inline package.json]: I think the project name must be `ccd-case-activity-api`

File: package.json [hunk 2]
```diff
@@ -9,24 +9,27 @@
     "test:end2end": "mocha test/e2e --timeout 15000",
```
Comment (tzarsmango) [inline package.json]: test command doesn't generate coverage report, so (I guess) running sonar scan does not generate a report with coverage

Comment (mario-paniccia) [reply package.json]: I would ignore this for now

File: README.md [hunk 3]
```diff
@@ -62,7 +62,7 @@ $ export NODE_ENV=dev
 $ export DEBUG='ccd-case-activity-api:*'
 $ yarn start
 
-> ccd-case-activity-api@0.0.1 start /Users/fatiho/Documents/hmcts/ws-webstorm/ccd-case-activity-api
+> ccd-case-activity-api@0.0.2 start /Users/fatiho/Documents/hmcts/ws-webstorm/ccd-case-activity-api
 > node ./bin/www
```
Comment (adr1ancho) [inline README.md]: What does notation > ccd-case-activity-api@0.0.2 mean?

**Comment by agilks** [comment]
One last try to get checks build to pass

File: README.md (hunk 3 above)
Comment (mario-paniccia) [reply README.md]: I think it is a copy and paste from fatih's shell prompt :)

Comment (mario-paniccia) [reply README.md]: I think is a copy and paste of fatih's shell prompt when he wrote the readme

**Pull request merged**